homeassistant/components/roku/quality_scale.yaml

```diff
@@ -0,0 +1,102 @@
+rules:
+  # Bronze
+  action-setup:
+    status: exempt
+    comment: |
+      The integration only has an entity service, registered in the media_player platform.
+  appropriate-polling:
+    status: done
+    comment: |
+      This integration uses 10 seconds as its default polling rate. Apps and Channels are fetched every 15 minutes.
+  brands: done
+  common-modules: done
+  config-flow-test-coverage: done
+  config-flow: done
+  dependency-transparency: done
+  docs-actions: done
+  docs-high-level-description: done
+  docs-installation-instructions: done
+  docs-removal-instructions: done
+  entity-event-setup:
+    status: exempt
+    comment: |
+      Entities of this integration use coordinator and do not explicitly subscribe to events.
+  entity-unique-id:
+    status: done
+    comment: |
+      Entities of this integration have a unique id based on the device serial number and entity description key.
+  has-entity-name: done
+  runtime-data: done
+  test-before-configure: done
+  test-before-setup: done
+  unique-config-entry:
+    status: done
+    comment: |
+      Config entries of this integration have an unique id based on the device serial number.
+
+  # Silver
+  action-exceptions:
+    status: exempt
+    comment: |
+      The integration only has an entity service.
+  config-entry-unloading: done
+  docs-configuration-parameters: done
+  docs-installation-parameters: done
+  entity-unavailable:
+    status: done
+    comment: This integration uses a coordinator.
+  integration-owner: done
+  log-when-unavailable:
+    status: done
+    comment: This integration uses a coordinator.
+  parallel-updates: done
+  reauthentication-flow:
+    status: exempt
+    comment: |
+      This integration does not require authentication.
+  test-coverage: done
```

Comment:
Instead of calling the entity register to enable the select entity in `test_application_state`, use the `entity_registry_enabled_by_default` test fixture

Reply:
Can we also use the freezer here?

Reply:
will address in #133624